ftw/file/imaging.py

```diff
@@ -0,0 +1,47 @@
+from Products.Archetypes.utils import shasattr
+from ftw.file.interfaces import IFtwFileField
+from plone.app.blob.mixins import ImageFieldMixin
+from plone.app.imaging.traverse import DefaultImageScaleHandler
+from zope.component import adapts
+from zope.interface import implementsOnly
+
+
+class FtwFileImageScaleHandler(DefaultImageScaleHandler):
+    adapts(IFtwFileField)
+
+    def getScale(self, instance, scale):
+        if self.context.get(instance).content_type.startswith('image/'):
+            return super(FtwFileImageScaleHandler, self).getScale(instance,
+                                                                  scale)
+        else:
+            return None
+
+
+class ImagingMixin(ImageFieldMixin):
+    """The Goal of this Mixin is to get the funtionality and benefits of an
+    Imagefield while not restricting the current funtionality of the field.
+    Therefore we Inherit from the ImageFieldMixin and remove its implemented
+    Interfaces since they are often used to restrict certain behaviours.
```

Comment:
- some words should be lowercase, see https://www.englishclub.com/writing/caps0.htm :wink: 
- `Imagefield` -> `ImageField` ? (assuming it's a class)
- We actually specifically want to remove the `IImageField`, right? What's the actual reason for this? Which bug / behavior do we want to prevent? I think the description is still to generic. For deciding later (e.g. in a year) whether we could remove / change that because of any circumstance or influence we'd have to be able to create a test case to verify that the problem does not occur; what would we test? Can we give some pointers here?